### PR TITLE
use count_bytes_reader from plugin-sdk-go

### PR DESCRIPTION
**Pull request by gabor**
we moved  `count_bytes_reader` to `plugin-sdk-go` in https://github.com/grafana/grafana-plugin-sdk-go/pull/1065 , this PR changes `grafana/grafana` to use it.

### CODE REVIEW

**Review by marefr — COMMENTED** [review]
Isn't there some additional go modules (go.mod) that needs an update? `go work sync`

Update: `make update-workspace` seems to be the preferred command

**Comment by gabor** [comment]
@marefr 
> Isn't there some additional go modules (go.mod) that needs an update? `go work sync`
> 
> Update: `make update-workspace` seems to be the preferred command

oh, thanks for the info, didn't know this command existed. i ran it and added the results.

**Review by marefr — APPROVED** [review]
LGTM

File: pkg/tsdb/grafana-postgresql-datasource/testdata/table/timestamp_convert_bigint.golden.jsonc [hunk 1]
```diff
@@ -14,9 +14,9 @@
 //  | Labels:                              | Labels:                           | Labels:             | Labels:                           |
 //  | Type: []*time.Time                   | Type: []*time.Time                | Type: []*int64      | Type: []*time.Time                |
 //  +--------------------------------------+-----------------------------------+---------------------+-----------------------------------+
-//  | 2023-12-21 12:21:27 +0000 UTC        | 2023-12-21 12:21:27 +0000 UTC     | 1703161287          | 2023-12-21 12:21:52 +0000 UTC     |
-//  | 2023-12-21 12:21:27.724 +0000 UTC    | 2023-12-21 12:21:27.724 +0000 UTC | 1703161287724       | 2023-12-21 12:21:52.522 +0000 UTC |
-//  | 2023-12-21 12:21:27.724919 +0000 UTC | 2023-12-21 12:21:27.724 +0000 UTC | 1703161287724919000 | 2023-12-21 12:21:52.522 +0000 UTC |
+//  | 2023-12-21 12:21:27 +0000 UTC        | 2023-12-21 07:21:27 -0500 EST     | 1703161287          | 2023-12-21 07:21:52 -0500 EST     |
```
Comment (kylebrandt) [inline pkg/tsdb/grafana-postgresql-datasource/testdata/table/timestamp_convert_bigint.golden.jsonc:17]: Hmm... need to run as UTC I think, or revert these lines.

**Pull request merged**